### DISCUSSION TITLE
PUI breadcrumb levels

### DISCUSSION THREAD

**Discussion by matmair** [inventree/InvenTree#7962] [Ideas]
PUI uses different approaches to breadcrumb levels right now. Most breadcrumbs do not include the item that is currently viewed. Some do. What approach is/should be canonical?

Manufacturer (current item not included)
![grafik](https://github.com/user-attachments/assets/8e5f5cdd-28b4-4668-8a1c-92642c42365a)
Build  (current item included)
![grafik](https://github.com/user-attachments/assets/626055af-e8b3-479a-9b7a-ba354a0ee617)

I am happy to fix whatever needs adjustment but unsure which is the right approach.

**Comment by SchrodingersGat** [comment]
I would vote for not including the current page.

@wolflu05 any strong preference on this?

**Reply by wolflu05** [reply]
Not really, just consistent
